src/array/nth/index.js

```diff
@@ -0,0 +1,20 @@
+function nth (array, index) {
+  if (!Array.isArray(array)) {
+    return undefined
+  }
+
+  if (index < 0) {
```

Comment:
What if arr.length - 10, but index = -50 ?
You should run this code after the next check. "if (index > array.length) {"
But use Math.abs there

Reply:
expect(nth([1, 2, 3, 4, 5], -10)).toEqual(4)